api/router.go

```diff
@@ -149,6 +149,8 @@ func NewRouter(s *Server) *mux.Router {
 		r.Methods("POST").Path("/v1/backingimages/{name}").Queries("action", name).Handler(f(schemas, action))
 	}
 
+	r.Methods("GET").Path("/v1/supportbundles/{bundleName}/download").Handler(f(schemas, s.SupportBundleDownload))
```

Comment:
With the new API, how does UI initiate the support bundle generation?

Reply:
@yasker  add the original http handler back. so no ui change is needed.